### PR TITLE
Idea browse page

### PR DESCRIPTION
**Pull request by bcbrennecke**
Not compatible for IE.  We will work on that.  Approve me!  @Mctalian @YashdalfTheGray

### CODE REVIEW

File: src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html [hunk 1]
```diff
@@ -1,12 +1,16 @@
-<md-card>
-     <md-card-content layout="row">
-    <div flex="85" layout="column">
-        <div><strong>{{idea.title}}</strong></div>
-        <div class="accent-text">{{idea.author}}</div>
+<md-card aria-label="insert Drive">
+<md-tooltip md-visible="demo.showTooltip" layout-wrap>
+{{idea.abstract}}
```
Comment (YashdalfTheGray) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: Nitpicking readability here but missing an indent

File: src/homeView/homeView.tpl.html [hunk 2]
```diff
@@ -14,8 +14,8 @@
             <center><h2>Innovate</h2></center>
             <p>Create an innovative idea and post it to Rok-Starter.  This idea will be visible to other innovators around the world.</p>
         </div>
-        <div flex flex-order="2">
-             <center><img src="assets/socialnetwork12.png" ng-click="navToBrowse()"></center>
+        <div flex flex-order="2" ng-click="navToBrowse()">
+             <center><img src="assets/socialnetwork12.png" ></center>
             <center><h2>Collaborate</h2></center>
```
Comment (YashdalfTheGray) [inline src/homeView/homeView.tpl.html]: This line and the line after it is indented 3 spaces instead of 4

File: src/homeView/homeView.tpl.html [hunk 3]
```diff
@@ -14,8 +14,8 @@
             <center><h2>Innovate</h2></center>
             <p>Create an innovative idea and post it to Rok-Starter.  This idea will be visible to other innovators around the world.</p>
         </div>
-        <div flex flex-order="2">
-             <center><img src="assets/socialnetwork12.png" ng-click="navToBrowse()"></center>
+        <div flex flex-order="2" ng-click="navToBrowse()">
+             <center><img src="assets/socialnetwork12.png" ></center>
```
Comment (YashdalfTheGray) [inline src/homeView/homeView.tpl.html]: This line is indented an extra space.

File: src/ideas/ideaBrowseView/ideaBrowseView.tpl.html [hunk 4]
```diff
@@ -6,19 +6,18 @@
...
+        <div flex="10"></div>
+        <div flex="80">
 
-    <div layout="column" flex>
-        <div flex>
-            <md-list class="top-ideas-list">
-                <md-subheader class="md-accent md-hue-1">Top Ideas</md-subheader>
+        
+    <div layout="row" flex layout-wrap>
                 <md-item ng-repeat="idea in topIdeas | search:searchText">
```
Comment (YashdalfTheGray) [inline src/ideas/ideaBrowseView/ideaBrowseView.tpl.html]: The `md-item` tag is indented too much.

File: src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html [hunk 5]
```diff
@@ -1,12 +1,16 @@
...
+</md-tooltip>
+<md-card-content layout="column">
+<div flex="85" layout= "column">
+    <div><strong>{{idea.title}}</strong></div>
+    <div class="accent-text">{{idea.author}}</div>
+</div>
+<div flex layout="row" class="extra-padding-left-md">
+    <div layout="row">
+        <span>{{idea.likes}} <a class = "idea-likes-click" ng-click="openLikes($event,idea.Likes)">Likes</a> {{idea.backs.length}} Backs and 0 People</span>
```
Comment (YashdalfTheGray) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: I assume that this line shows the likes and the backs. There is a "and 0 people" string at the end of this line, what's that about?

Comment (Mctalian) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: I assume that one is stubbed for team

Comment (YashdalfTheGray) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: Oh I see. Makes sense.

File: src/ideas/ideaBrowseView/ideaBrowseView.tpl.html [hunk 6]
```diff
@@ -6,19 +6,18 @@
         <label>Search</label>
         <input name="search" ng-model="searchText">
     </md-input-container>
+    <md-subheader class="md-accent md-hue-1">Top Ideas</md-subheader>
```
Comment (Mctalian) [inline src/ideas/ideaBrowseView/ideaBrowseView.tpl.html]: I don't think we need this subheader in this view. That's only really necessary in the sidenav

**Comment by Mctalian** [comment]
I actually really like the mouseover blurb. I didn't think I was going to, but the animation makes it feel very smooth.

File: src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html [hunk 7]
```diff
@@ -1,12 +1,16 @@
-<md-card>
-     <md-card-content layout="row">
-    <div flex="85" layout="column">
-        <div><strong>{{idea.title}}</strong></div>
-        <div class="accent-text">{{idea.author}}</div>
+<md-card aria-label="insert Drive">
+<md-tooltip md-visible="demo.showTooltip" layout-wrap>
+{{idea.abstract}}
+</md-tooltip>
+<md-card-content layout="column">
```
Comment (Mctalian) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: Take out `layout="column"` and you will be IE compatible :)

File: src/ideas/ideaBrowseView/ideaBrowseView.tpl.html [hunk 8]
```diff
@@ -6,19 +6,18 @@
...
+    <div layout="row" layout-align="center center">
+        <div flex="10"></div>
```
Comment (Mctalian) [inline src/ideas/ideaBrowseView/ideaBrowseView.tpl.html]: If you actually want an extra "10" padding, use `offset="10"` on the next line: `<div flex="80" offset="10">`

Honestly though, without the offset, I think it looks fine. I would even argue that `flex="90"` might be more desirable. Your all. Either way, remove this blank flex 10 div.

File: src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html [hunk 9]
```diff
@@ -1,12 +1,16 @@
...
+<div flex="85" layout= "column">
+    <div><strong>{{idea.title}}</strong></div>
+    <div class="accent-text">{{idea.author}}</div>
+</div>
+<div flex layout="row" class="extra-padding-left-md">
+    <div layout="row">
```
Comment (Mctalian) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: Unnecessary. `<span>` by default will provide "row-like" behavior.

File: src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html [hunk 10]
```diff
@@ -1,12 +1,16 @@
...
+<md-card aria-label="insert Drive">
+<md-tooltip md-visible="demo.showTooltip" layout-wrap>
+{{idea.abstract}}
+</md-tooltip>
+<md-card-content layout="column">
+<div flex="85" layout= "column">
+    <div><strong>{{idea.title}}</strong></div>
+    <div class="accent-text">{{idea.author}}</div>
+</div>
+<div flex layout="row" class="extra-padding-left-md">
```
Comment (Mctalian) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: Did you intend to have `class="extra-padding-left-md"` here?

If so, add the class to the `<span>` instead. Either way, remove this whole div.

File: src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html [hunk 11]
```diff
@@ -1,12 +1,16 @@
-<md-card>
-     <md-card-content layout="row">
-    <div flex="85" layout="column">
-        <div><strong>{{idea.title}}</strong></div>
-        <div class="accent-text">{{idea.author}}</div>
+<md-card aria-label="insert Drive">
```
Comment (Mctalian) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: I think your aria-label is a copy past. Can you instead make it `{{idea.title}} by {{idea.author}}`

aria-label is used for screen readers for people with visual disabilities. If the thing said "insert Drive" I think they'd be very confused.

File: src/ideas/ideaBrowseView/ideaBrowseView.tpl.html (hunk 4 above)
Comment (Mctalian) [inline src/ideas/ideaBrowseView/ideaBrowseView.tpl.html]: The `md-item` tag also appears to have been deprecated. You used to use it with `md-list` which we're not using here, so you can move the `ng-repeat` into the `idea-item-browse` and remove the `md-item`.

File: src/ideas/ideaBrowseView/ideaBrowseView.tpl.html [hunk 12]
```diff
@@ -6,19 +6,18 @@
...
+    <div layout="row" flex layout-wrap>
                 <md-item ng-repeat="idea in topIdeas | search:searchText">
                     <idea-item-browse idea="idea" ui-sref="idea({ ideaId: idea.id })" class="idea-pointer"></idea-item-browse>
                 </md-item>
-            </md-list>
-            <md-divider></md-divider>
-            <!-- <div layout="row"><md-button class="md-primary" ng-click="navTo('home')" flex>Home</md-button></div> -->
-            <!-- <div layout="row"><md-button class="md-primary" ng-click="navTo('idea')" flex>Idea (Test only)</md-button></div> -->
-        </div>
+        <md-divider></md-divider>
```
Comment (Mctalian) [inline src/ideas/ideaBrowseView/ideaBrowseView.tpl.html]: I don't think this is needed anymore.

File: src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html (hunk 5 above)
Comment (bcbrennecke) [inline src/ideas/ideaItemBrowse/ideaItemBrowse.tpl.html]: yes.

**Comment by Mctalian** [comment]
Approved! :+1:

**Comment by YashdalfTheGray** [comment]
Approved!

**Pull request merged**